src/drv_oculus_rift_s/rift-s.c

```diff
@@ -55,8 +55,10 @@ static rift_s_hmd_t *find_hmd(char *hid_path)
 	device_list_t* current = rift_hmds;
 
 	while (current != NULL) {
-		if (strcmp(current->path, hid_path)==0)
+		if (strcmp(current->path, hid_path)==0) {
+			current->hmd->use_count++;
```

Comment:
Would it be better to do this in open_device(), after the last possible error return?

Reply:
It's cleaner to keep it in find_hmd, and make sure to call release_hmd() in the 1 early error return in open_device() - otherwise one also has to change open_hmd() to not set the initial use_count to 1, or something more complicated in open_device(). This way is symmetrical.

Good catch.